src/Index.ts

```diff
@@ -1,5 +1,6 @@
 import type { GuildMember, Message, MessageReaction, User } from "discord.js";
```

Comment:
has this file always been named capital 'I' Index.ts 💀

Reply:
Yes sirrie

Reply:
Yes. This has irked me for a while.